feature/platform/fabric/otg_tests/sampled_backplane_capacity_counters_test/README.md

```diff
@@ -32,26 +30,10 @@ the available backplane capacity counters correctly while forwarding traffic
 
     *   /components/component/integrated-circuit/backplane-facing-capacity/state/consumed-capacity
 
-    *   Initiate traffic:
-
-        *   Initiate traffic as per below threshold:
-    
-             Port number   | Interface1(line rate %)
-            -------------- | -----------------------
-            Port1          | 20%
-            Port2          | 20%
+        *   consumed-capacity is the sum of the admin-up front panel interface speeds connected to the integrated-circuit
 
     *   Validate that we are receiving consumed capacity metrics at the selected SAMPLE interval
```

Comment:
since we are now using ON_CHANGE, we need to subscribe, set a port enabled=false, observe the change, set a port enabled=true, observe the change.